OpenRA.Mods.Common/Traits/CashTrickler.cs

```diff
@@ -87,7 +90,15 @@ void AddCashTick(Actor self, int amount)
 
 		void ModifyCash(Actor self, Player newOwner, int amount)
 		{
-			amount = resources.ChangeCash(amount);
+			// Should the cash trickler use resource storage
```

Comment:
Imho this comment is not required, since `UseStorage` should be self-explanatory (we could rename it to `UseResourceStorage` if you think that makes it clearer).